### ISSUE TITLE
Building breaks with env settings in project on Windows.

### ISSUE THREAD

**Issue by EraYaN** [SublimeText/LaTeXTools#749]
After the latest update.

The following:
```json
"builder_settings" : {
	"windows" : {
		"env": {
			"BIBTEX": "biber"
		}
	}
},
```
in the settings part of the project file breaks compilation completely. If I remove it it builds correctly (without biber though)

```
[Compiling ..\docs\thesis\thesis.tex]
```
is the only output that is displayed.

Using latest MikTex 64-bit, Sublime 3114, LaTeXTools (latest as of now)

**Comment by ig0774** [comment]
What shows up on the console (`View | Console`)?

**Comment by EraYaN** [comment]
```
<class '_latextools_traditionalBuilder.TraditionalBuilder'>
2
Welcome to thread Thread-7
Exception in thread Thread-7:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "C:\Users\Erwin\AppData\Roaming\Sublime Text 3\Packages\LaTeXTools\makePDF.py", line 90, in run
    os.environ.update(self.caller.env.items());
  File "./python3.3/collections/abc.py", line 579, in update
  File "./python3.3/os.py", line 661, in __setitem__
  File "./python3.3/os.py", line 713, in check_str
TypeError: str expected, not SettingsWrapper
```

**Comment by ig0774** [comment]
@EraYaN: This is fixed in v3.8.2, which should be available in the next hour. Thanks for the bug report!

**Comment by EraYaN** [comment]
Thanks, I can confirm that it works.
